aksel.nav.no/website/pages/eksempler/actionmenu/links.tsx

```diff
@@ -0,0 +1,67 @@
+import Link from "next/link";
+import { ChevronDownIcon } from "@navikt/aksel-icons";
+import { ActionMenu, Button } from "@navikt/ds-react";
+import { withDsExample } from "@/web/examples/withDsExample";
+
+const Example = () => {
+  return (
+    <ActionMenu defaultOpen>
+      <ActionMenu.Trigger>
+        <Button
+          variant="secondary-neutral"
+          icon={<ChevronDownIcon aria-hidden />}
+          iconPosition="right"
+        >
+          Meny
+        </Button>
+      </ActionMenu.Trigger>
+      <ActionMenu.Content>
+        <ActionMenu.Group label="See code for implementation">
+          <ActionMenu.Item as="a" href="#">
+            # lenke
+          </ActionMenu.Item>
+          <Link href="#123" passHref legacyBehavior>
+            <ActionMenu.Item as="a">Nextjs Pages router</ActionMenu.Item>
+          </Link>
+          <ActionMenu.Item>
+            Nextjs App router (se kommentert kode)
+          </ActionMenu.Item>
```

Comment:
Do we have an example of Nextjs App router?

Reply:
Removed it, same implementation for pages and app router 🤞